inheritance1/PercentDiscountPurchase.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class PercentDiscountPurchase extends Purchase{
+    private final static int MINIMUM = 3;

```

Comment:
Not a good name for a constant, I think, even if it is private and only for this class